src/lib/support/logging/Constants.h

```diff
@@ -53,6 +53,7 @@ enum LogModule
     kLogModule_SetupPayload,
     kLogModule_AppServer,
     kLogModule_Discovery,
+    kLogModule_Platform,
```

Comment:
All code in `src/platform` uses/should use `DeviceLayer` logging module. Please don't introduce another module.

Reply:
Done.